app/javascript/dashboard/components/widgets/conversation/bubble/Actions.vue

```diff
@@ -113,6 +127,13 @@ export default {
   }
 }
 
+.right {
+  .ion-reply,
+  .ion-android-open {
```

Comment:
Can we target with a generic class name like, `action-icons` or something ? So we don't have to rely on ion classes.

Reply:
This makes sense, I will create a separate PR for the same, missed this review.